modules/i2c/i2c.h

```diff
@@ -12,7 +12,9 @@
 #include <avr/io.h>
 #include <avarix/intlvl.h>
 #include <stdint.h>
+#include <stddef.h>
```

Comment:
The use of NULL keyword in [slavex.inc.c](https://github.com/robotter/avarix/blob/dev/modules/i2c/slavex.inc.c) file.
It has been moved in [slavex.inc.c](https://github.com/robotter/avarix/blob/dev/modules/i2c/slavex.inc.c) in ef2ed196d4af7040c2ddbc8e253721181ca7abdd